modules/masterbar/class-admin-menu.php

```diff
@@ -210,13 +210,14 @@ public function set_site_card_menu_class( array $menu ) {
 			}
 
 			$domain  = wp_parse_url( get_home_url(), PHP_URL_HOST );
-			$classes = 'toplevel_page_site-card';
+			$classes = ' toplevel_page_site-card';
 
-			if ( ( function_exists( 'blavatar_exists' ) && blavatar_exists( $domain ) ) || has_site_icon() ) {
+			// webclip.png is the default on WoA sites. Anything other than that means we have a custom site icon.
+			if ( ( function_exists( 'blavatar_exists' ) && blavatar_exists( $domain ) ) || ( has_site_icon() && 'https://s0.wp.com/i/webclip.png' !== get_site_icon_url( 512 ) ) ) {
```

Comment:
Is this going to work in simple sites?

Reply:
Simple will use the `blavatar_exists()` check